airbyte-cdk/python/unit_tests/sources/file_based/scenarios/validation_policy_scenarios.py

```diff
@@ -459,50 +459,12 @@
         "read": [
             {
                 "level": "WARN",
-                "message": "Stopping sync in accordance with the configured validation policy. Records in file did not conform to the schema. stream=stream1 file=a/a1.csv validation_policy=wait_for_discover n_skipped=0",
+                "message": "Stopping sync in accordance with the configured validation policy. Records in file did not conform to the schema. stream=stream1 file=a/a1.csv validation_policy=ValidationPolicy.wait_for_discover n_skipped=0",
```

Comment:
Nit: can we make this 
```suggestion
                "message": 'Stopping sync in accordance with the configured validation policy. Records in file did not conform to the schema. stream=stream1 file=a/a1.csv validation_policy="Wait for Discover" n_skipped=0',
```

Reply:
done!